### ISSUE TITLE
Rescue Raiders' speech not working

### ISSUE THREAD

**Issue by tomcw** [a2-4am/4cade#365]
My setup:
- Total Replay v4.01.hdv
- AppleWin 1.29.16.0 (latest release)
- Either Mockingboard in slots 4&5 or Phasor in slot-4

When I start TR, it reports "and it talks!" ...and speech works with Berzap! and Bejeweled.

But for Rescue Raiders, when it boots it only shows "64KR1.5" it the bottom-left of the TEXT screen, when it should overwrite this with "64KMCK4" if it detects a Mockingboard in slot-4.

I reverted to "Total Replay v4.0.hdv" and that also fails.
But on reverting to "Total Replay v4.0-alpha.2.2mg" speech works correctly for Rescue Raiders.

**Comment by tomcw** [comment]
There's a missing `SEC` before the LDA (zp),y; SBC (zp),y for the 2nd 6522 at $Cn80...

```
4109: sec
 ldy #$00
 lda ($72),y  ; ($72) = $C404
 sbc ($72),y
 jmp 37e
```

It looks like the BCC is out by 1, and skips the SEC here:
```
37E: sec
 sbc #$05
 cmp #$02
 bcc 389  ; should this be bcc 388 to include "sec" ?
 jmp 4144
388: sec
389: jmp 4114
```

```
4114: ldy #$80
 lda ($72),y  ; ($72) = $C404
 sbc ($72),y
 jmp 38c
```

**Comment by peterferrie** [comment]
Thanks.  You are correct.  Fixed in 7984f3a916b8932b0a68c1d24727732244e07f99.
